flink-connector-aws-base/src/test/java/org/apache/flink/connector/aws/testutils/AWSServicesTestUtils.java

```diff
@@ -114,6 +116,14 @@ public static void createBucket(S3Client s3Client, String bucketName) {
         }
     }
 
+    public static void createSqs(String sqsName, SqsClient sqsClient) {
```

Comment:
This shouldn't be in `aws-base` we could move it to `SqsTestUtils`

Reply:
Moved to SqsTestUtils